langchain/tools/arxiv/tool.py

```diff
@@ -22,3 +27,12 @@ def _run(self, query: str) -> str:
     async def _arun(self, query: str) -> str:
         """Use the Arxiv tool asynchronously."""
         raise NotImplementedError("ArxivAPIWrapper does not support async")
+
+
+
+llm = OpenAI(temperature=0)
```

Comment:
We shouldn't be running code like this on the outer level or importing the agents into the `tools/` directory :)

Reply:
from langchain.utilities.arxiv import ArxivAPIWrapper
from langchain.utilities import ArxivAPIWrapper

these lines don't work . The first is mentioned in github and the later on the webpage for langchain/agents/tools/arxiv

I am sorry I though I should just uploaded the end to end code for how to use it the arxiv api since it wasn't mentioned on the webpage or the github under utilities either

Reply:
Are you running a recent version of LangChain? The imports work fine right now

Reply:
Yeah , now the updated package has the working import , Thanks!